src/text-input/index.ts

```diff
@@ -27,16 +37,19 @@ export type TextInputType = 'text' | 'email' | 'number' | 'password' | 'search'
  * @property value           The current value
  */
 
-export interface TextInputProperties extends ThemedProperties, InputProperties, FocusProperties, LabeledProperties, PointerEventProperties, KeyEventProperties, InputEventProperties, CustomAriaProperties {
+export interface TextInputProperties extends ThemedProperties, Omit<InputProperties, 'invalid'>, FocusProperties, LabeledProperties, PointerEventProperties, KeyEventProperties, InputEventProperties, CustomAriaProperties {
```

Comment:
May well be better if you just inline the appropriate properties from `InputProperties` here. Probably going to phase out all these extended properties

Reply:
Done. I copied `InputProperties` into this interface and then removed the `Omit`.